content/organization/tech/security/index.en.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Security"
+menuTitle: "Security"
+weight: 54
+---
+
+As organizers you will be responsible for accounts such as email, social media accounts.
+Here are some steps that will help reduce the risk of someone's hacking into your account.
+
+## Use a personal password manager!
+
+For your personal use (for R-Ladies things and in general!), we strongly recommend using a personal password manager, be it a free one like KeePass (with a back-up you'd set up on Dropbox or Google Drive) or a paid service like 1Password.
+
+If you feel on the fence, ask other organizers or your friends about their experience with some password managers.
+
+### KeePass workflow
+
+* save all your passwords in a KeePass database.
+* have it backed-up on Dropbox or Google Drive.
+* only learn two passwords: the one to decrypt the KeePass database, the one to your Google Drive account.
+
+### 1Password workflow
+
+You can even share an account with family members.
+
+## If possible use two-factor authentication (2FA)
+
+You can secure accounts such as GitHub accounts, Twitter accounts with two-factor authentication (2FA).
+It means that to log in into the account you will need both the password and your phone where you will be given a temporary code either by text message or via an app such as DuoMobile.
+
+Make sure to save the recovery codes of 2FA in your personal password manager, and/or to share them with your team.
+
+Note that apps like DuoMobile allow you to sync your account with GoogleDrive which can be very handy when changing phones for instance.
+
+## Share passwords and 2FA recovery codes securely with co-organizers
+
+Your chapter will have a bunch of passwords and 2FA recovery codes to share.
+How to do that?
+
+* You might try and apply for 1Password OSS free plan.
```

Comment:
* You might try and apply for [1Password OSS free plan](https://github.com/1Password/1password-teams-open-source)